src/components/fx/hover.js

```diff
@@ -1236,3 +1406,12 @@ function hoverChanged(gd, evt, oldhoverdata) {
     }
     return false;
 }
+
+function spikesChanged(gd, oldspikepoints) {
+    // don't emit any events if nothing changed
```

Comment:
Looks like this got copied from `function hoverChanged() {}`, but spike lines don't emit any events whatsoever. Would you mind :hocho: this line to avoid confusion.

Reply:
I use it to avoid the redrawing of the plot because of new spikelines if spikelines points didn't change